test/unit/Cardano/Wallet/Primitive/AddressDerivationSpec.hs

```diff
@@ -228,7 +232,12 @@ goldenYoroiAddr (seed, recPwd) cc accIx addrIx addr =
         accXPrv = deriveAccountPrivateKey encPwd rootXPrv accIx
         addrXPrv = deriveAddressPrivateKey encPwd accXPrv cc addrIx
     in
-        fmt (build $ keyToAddress $ publicKey addrXPrv) === addr
+        fmt (build $ yoroiKeyToAddress $ publicKey addrXPrv) === addr
+  where
+    yoroiKeyToAddress xpub =
```

Comment:
🤔

Had to do this to have the tests pass, as the Yoroi addresses don't seem to contain the protocol magic.

Opted to not make the existing `keyToAddress` more flexible for simplicity.

Reply:
Yeah, those addresses are from `mainnet` I suppose. Ideally, we would change the golden address above with a Yoroi one from testnet with the same derivation attributes (you can do that using `cardano-cli wallet address --protocol-magic ... `)

Reply:
Would rather do in separate pr, maybe same time as for the signed tx golden

Reply:
Hmmm.. Okay.